### PR TITLE
Hotfix/missing sftags

### PR DESCRIPTION
**Pull request by Edholm**

### CODE REVIEW

File: backend/src/main/kotlin/rocks/didit/sefilm/domain/dto/CalendarEventDTO.kt [hunk 1]
```diff
@@ -10,19 +10,16 @@ data class CalendarEventDTO(val summary: String,
...
-                    mapOf("dateTime" to end.toLocalDateTime().toString(), "timeZone" to end.zone)
-            )
-
-        }
-
+      return CalendarEventDTO(
+        summary = summary,
+        location = location?.name ?: "N/A",
+        attendees = emails.map { mapOf("email" to it) },
+        start = mapOf("dateTime" to start.toOffsetDateTime().toString()),
```
Comment (Dr-Horv) [inline backend/src/main/kotlin/rocks/didit/sefilm/domain/dto/CalendarEventDTO.kt]: Funkar det här? Är rätt säker på att det var tvungen att vara utan tidszon i formatet? Men jag kanske läste dokumentationen slarvigt?

File: backend/src/main/kotlin/rocks/didit/sefilm/web/controllers/ShowingController.kt [hunk 2]
```diff
@@ -89,36 +85,28 @@ class ShowingController(private val repo: ShowingRepository,
...
   fun createGoogleCalendarEvent(@PathVariable id: UUID, @RequestBody body: List<String>): ResponseStatusDTO {
-    val showing = repo.findById(id)
-      .map { showing ->
-        if (!showing.isLoggedInUserAdmin()) throw AccessDeniedException("Only the admin can create gcal events")
-        showing
-      }
-      .orElseThrow { NotFoundException("showing '$id") }
+    val showing = findOne(id)
+    if (!showing.isLoggedInUserAdmin()) throw AccessDeniedException("Only the admin can create calendar events")
+    if (!showing.calendarEventId.isNullOrBlank()) throw BadRequestException("Calendar event already created")
```
Comment (Dr-Horv) [inline backend/src/main/kotlin/rocks/didit/sefilm/web/controllers/ShowingController.kt]: Ah, smart

File: backend/src/main/kotlin/rocks/didit/sefilm/domain/dto/CalendarEventDTO.kt (hunk 1 above)
Comment (Edholm) [inline backend/src/main/kotlin/rocks/didit/sefilm/domain/dto/CalendarEventDTO.kt]: "The time, as a combined date-time value (formatted according to RFC3339). A time zone offset is required unless a time zone is explicitly specified in timeZone." per the documentation. It worked locally on my machine at least. 

I encourage you to try on your machine as well :)

**Comment by Edholm** [comment]
#116 borde mergeas före denna.

**Pull request merged**